honeybee_3dm/grid.py

```diff
@@ -0,0 +1,69 @@
+"""Import Rhino surfaces as Honeybee grid objects from a Rhino3DM file."""
+import warnings
+# The Rhino3dm library provides the ability to access content of a Rhino3dm
+# file from outside of Rhino
+import rhino3dm
+
+# Importing Honeybee dependencies
+from honeybee_radiance.sensorgrid import SensorGrid
+from honeybee.typing import clean_and_id_string, clean_string
+
+# Importing dependencies from Honeybee-3dm package
+from .togeometry import brep_to_face3d, mesh_to_mesh3d, check_planarity
+from .helper import filter_objects_by_layer
+
+
+# TODO: expose an option to change the target layer name
+# TODO: Grid size and offset will have to provided using the config file
+
+
+def import_grids(rhino3dm_file, tolerance, grid_size=1, grid_offset=0):
+    """Creates Honeybee grids from a rhino3dm file.
+
+    This function assumes all the grid objects are under a layer named ``grid``.
+
+    Args:
+        rhino3dm_file: The rhino file from which Honeybee grids will be created.
+        tolerance: A rhino3dm tolerance object. Tolerance set in the rhino file.
+        grid_size: Grid size. Defaults to 1.
+        grid_offset: Grid offset from the grid geometry. Defaults to 0.
+
+    Returns:
+        A list of Honeybee grids.
+    """
+
+    # Objects from rhino file
+    try:
+        grid_objs = filter_objects_by_layer(rhino3dm_file, 'grid')
+        warnings.warn(
+            'Only objects on layer "grid" will be used to create grids.')
+    except ValueError:
+        # no layer named grid
+        return []
+
+    hb_grids = []
+
+    for obj in grid_objs:
+        geo = obj.Geometry
+        # If it's a Brep
+        if isinstance(geo, rhino3dm.Brep):
+            # Check if the Brep is planar
+            if check_planarity(geo):
+                face3d = brep_to_face3d(geo, tolerance)[0]
```

Comment:
Shouldn't you be creating a mesh grid for all of the `Face3Ds` that this method returns?  You can create multiple Meshes and then join them using the method here:
https://www.ladybug.tools/ladybug-geometry/docs/ladybug_geometry.geometry3d.mesh.html#ladybug_geometry.geometry3d.mesh.Mesh3D.join_meshes

Reply:
I revised the brep_to_face3d method to return a single Ladybug Face3D object. I realize it was creating confusion as to why I am using just the first Face3d object.